serializers/src/glTF/2.0/glTFUtilities.ts

```diff
@@ -193,4 +193,22 @@ export class _GLTFUtilities {
             tangent.z /= length;
         }
     }
+
+    public static _NormalizeMatricesWeightsFromRef(matricesWeights: Vector4) {
+        const sum = matricesWeights.x + matricesWeights.y + matricesWeights.z + matricesWeights.w;
+        if (sum > 0) {
+            matricesWeights.scaleInPlace(1 / sum);
+        }
+    }
+
+    public static _GetRightHandedMatrixFromRef(matrix: Matrix) {

```

Comment:
I'm not too sure how to derive this change, I came to this result visually after noticing that `Matrix.toggleModelMatrixHandInPlace`
https://github.com/BabylonJS/Babylon.js/blob/a11d765c5dfa6d6db2e3945ac84389a22d2ca9d1/src/Maths/math.vector.ts#L4327-L4338

inverted m[9] and m[14] of my output inverse bind matrix, so I created this custom function to handle the matrix handedness converstion, but I'm not sure if this is 100% correct...